disable returning to the episodes menu if only one episode is present

rfomin opened the pull request:
Fix this issue: [[doomworld]](https://www.doomworld.com/forum/topic/112333-this-is-woof-1130-july-06-2023/?do=findComment&comment=2676304)

fabiangreffrath commented:
I think I remember we fixed this before. Wasn't there even a poll whether to show the episode choice menu or not when there was only a single episode available?

rfomin commented:
> I think I remember we fixed this before

Yes, we've fixed this, but we missed one bug: it still shows up when the user presses the backspace key on the skill selection screen.

fabiangreffrath commented:
So many pitfalls. 😁
